docs/CHANGELOG.md

```diff
@@ -66,6 +69,8 @@ it is attached to. ([#1282])
 [#1404]: https://github.com/amethyst/amethyst/pull/1404
 [#1408]: https://github.com/amethyst/amethyst/pull/1408
 [#1405]: https://github.com/amethyst/amethyst/pull/1405
+[#1411]: https://github.com/amethyst/amethyst/pull/1411
+[#1412]: https://github.com/amethyst/amethyst/pull/1412
```

Comment:
The entry for 1412 seems to be missing.

Reply:
* Added optional feature gates which will reduce compilation times when used. ([#1412])